include/dd/Package.hpp

```diff
@@ -1986,7 +2034,7 @@ namespace dd {
         }
 
         CMat getMatrix(const mEdge& e) {
-            std::size_t dim = 1 << (e.p->v + 1);
+            std::size_t dim = 1ULL << (e.p->v + 1);
```

Comment:
Technically, the type is not correct, but either using `unsigned long long dim = ...` or `static_cast<std::size_t>(1) << ...` seem very ugly. Maybe keep this in mind if anybody reports problems when compiling and running on exotic platforms?

Same goes for instances below.

Reply:
On reasonable platforms `std::size_t` should be 64bit and `unsigned long long` should be at least that. As these function should most probably not be used for more than a couple of qubits, I would not see this as a problem.
Nevertheless, good to keep it in mind.